### ISSUE TITLE
Any srcipt to remove "sort" from top bar of database and propreties inside the page?

### ISSUE THREAD

**Issue by alerezaaa** [stephenou/fruitionsite#215]
Hi  
I hope I could write this on Discussion but its not open

How can I remove these parts from the page???  
 

![image](https://user-images.githubusercontent.com/86428901/205235909-27373719-d5dc-47ad-9619-99fab9027124.png)

  
On mobile, its kind of more visible

![](https://user-images.githubusercontent.com/86428901/205236848-09ef1c58-8c1f-419f-8057-49758542078d.png)

  
I want to remove only "Sort" and all of its components

and also inside the page:  
 

![image](https://user-images.githubusercontent.com/86428901/205236245-02b6623a-274c-4d2c-8a64-91d9c91fc1e8.png)

Any way or script to remove these parts from the page (by setting Notion or by using an script in Workers)

In Summary, make [this](https://live.alerezaa.ir) look like [this](https://super.alerezaa.ir) (the second site deployed using super.so but I dont really like it)

Thanks in advance

**Comment by colinroper** [comment]
I don't have a great answer for you but maybe this gets you started... 

Notion doesn't give class names or other identifiers to these elements so I am only able to find a hacky way of doing it. After further review, I am seeing that it works on the desktop but doesn't seem to affect mobile. I'm not able to easily poke around the mobile browser inspector to see why. So this is what I did to remove the **filter** on the desktop:

```
/* Hide sort and filter in gallery view*/
div.notion-collection_view-block > div > div > div.notion-focusable:nth-of-type(1) {display: none !important;}

/* Hide sort and filter submenu in gallery view*/
div.notion-scroller > div > div.notion-focusable  {display: none !important;}
div.notion-scroller > div.notion-focusable  {display: none !important;}
```

For you to remove the sort, you'll just want to change the `...nth-of-type(1)...` to `...nth-of-type(2)...`

If you find a way to remove the relevant section for mobile do let me know.

**Comment by alerezaaa** [comment]
> I don't have a great answer for you but maybe this gets you started...
> 
> Notion doesn't give class names or other identifiers to these elements so I am only able to find a hacky way of doing it. After further review, I am seeing that it works on the desktop but doesn't seem to affect mobile. I'm not able to easily poke around the mobile browser inspector to see why. So this is what I did to remove the **filter** on the desktop:
> 
> ```
> /* Hide sort and filter in gallery view*/
> div.notion-collection_view-block > div > div > div.notion-focusable:nth-of-type(1) {display: > none !important;}
>
> /* Hide sort and filter submenu in gallery view*/
> div.notion-scroller > div > div.notion-focusable  {display: none !important;}
> div.notion-scroller > div.notion-focusable  {display: none !important;}
> ```
>
> For you to remove the sort, you'll just want to change the `...nth-of-type(1)...` to `...nth-of-type(2)...`
>
> If you find a way to remove the relevant section for mobile do let me know.

Thank you so much, I'll take a look at it

**Comment by colinroper** [comment]
Poking around this a bit more... you can remove the entire bar with search/filter/sort for both web and mobile with this:
```
/* Hide sort and filter submenu in gallery view */
div.notion-scroller > div > div[role=button] {display: none !important;}
div.notion-scroller > div[role=button]  {display: none !important;}
```

**Comment by alerezaaa** [comment]
thank you so much

it solved the mobile view

**Comment by ViktorPika** [comment]
<img width="779" alt="Screenshot 2024-02-21 at 12 23 21" src="https://github.com/stephenou/fruitionsite/assets/57147513/810494c4-31e9-479d-b053-15a6f657c969">
 
 Will this work here too?
